samples/spring-valueentity-counter/src/main/java/com/example/CounterEntity.java

```diff
@@ -8,12 +8,12 @@
 import org.springframework.web.bind.annotation.RequestBody;
 
 @EntityType("counter")                                     // <1>
-@EntityKey("counter_id")                                   // <2>
-public class CounterEntity extends ValueEntity<Integer> {  // <3>
+public class CounterEntity extends ValueEntity<Integer> {  // <2>
 
   @Override
-  public Integer emptyState() { return 0; }                  // <4>
+  public Integer emptyState() { return 0; }                  // <3>
 
+  @EntityKey("counter_id")                                   // <4>
```

Comment:
Now that EntityKey can be on a method, it makes it easier to put it next to the PostMapping. At least for the getting started guide where we don't want to be too verbose.

Reply:
I like this. It separates, even more, the idea of 'service' from the 'entity'.